addons/mass_mailing/models/mail_mail.py

```diff
@@ -3,81 +3,74 @@
 
 import re
 import urlparse
-import re
 import werkzeug.urls
 
-from openerp import tools
-from openerp import SUPERUSER_ID
-from openerp.osv import osv, fields
+from odoo import tools
+from odoo import api, fields, models
 
 
 URL_REGEX = r'(\bhref=[\'"]([^\'"]+)[\'"])'
 
 
-class MailMail(osv.Model):
+class MailMail(models.Model):
     """Add the mass mailing campaign data to mail"""
     _name = 'mail.mail'
     _inherit = ['mail.mail']
```

Comment:
tremove _name. here _inherited object are same. so just _inherit = 'mail.mail' is enough.